docs/api-docs/catalog/v2-vs-v3.md

```diff
@@ -102,7 +102,7 @@ In V3, any variants or modifier adjusters created with non-null core properties
 * [Deprecations and sunsets](/api-docs/getting-started/deprecations-and-sunsets)
 
 ### Endpoints
```

Comment:
although these links currently represent endpoints, the link text describing them doesn't because an "endpoint" includes the HTTP method. Either the title of the section or the link text for the endpoints should change.

Reply:
Changed link texts to include HTTP verbs